app/storybook/documentation/ui-kit/provider/LogProvider.mdx

```diff
@@ -0,0 +1,27 @@
+import { Canvas, ArgTypes, Meta, Story, Source, Description, Primary, Controls, Stories } from '@storybook/blocks'
+import { prettier } from '../../../src/utils'
+import * as LogStories from '../../../../../packages/ui-kit/src/components/Logger/Logger.stories'
+
+<Meta title="PROVIDER/Log" />
+
+# Log Provider
+
+By using the Log Provider, you can provide your own messages. 
+You can enable different level of logs by setting the log level type such as "error", "warn", "info" or "debug". 
+You can also disable all logs by setting log level as "off". 
+
+Log Provider will enable you to provide more accurate feedback to users in case of technical errors, warnings, info or other issues.
+
+For example, when you set up log level as "error" to display an error log in the TimeSeries component such as "chart data cannot be loaded", you can use the following
+JavaScript code:
```

Comment:
I think we should rewrite this, focusing primarily on how users can configure the logging for components they already use. Most users will want to turn off logging, or change the log level to see more. How about this:

````suggestion
The Propel UI Kit logs all errors to the browser's console by default. You can configure this behavior using the LogProvider component. For example, if your app contains a Counter component like this:

```ts
import { Counter } from "@propeldata/ui-kit";

function App() {
  return (
    <Counter />
  );
}
```

you can nest it in a LogProvider component to change the log level to "warn":

```ts
import { Counter, LogLevel, LogProvider } from "@propeldata/ui-kit";

function App() {
  return (
    <LogProvider logLevel={LogLevel.Warn}>
      <Counter />
    </LogProvider>
  );
}
```

You can set the log level to "error", "warn", "info" or "debug". Additionally, you can disable all logs by setting log level to "off" (for example, in your production build).

Finally, we recommend configuring at most one LogProvider per app.
````